CHANGELOG.md

```diff
@@ -4,11 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v2.5.1] - 2019-01-29
+
+### Changed
+- Added option `bearer_schema` to the Faraday middleware to allow the caller to specify whether to follow the Bearer schema when setting the JWT token in the Authorization request header (defaults to false)
+
 ## [v2.5.0] - 2019-01-21
 
 ### Changed
+- Ensure we use the JWT Token in the Authorization header using the Bearer schema. We will still support Authorization headers without the Bearer schema
 
-- Ensure we use the JWT Token in the Authorization header using the Bearer schema. We will still support Authorization headers without the Bearer schema.
+### Breaking changes:
```

Comment:
I'm not sure if "breaking changes" is the right heading here as this is a minor/patch release so this essentially shouldn't exist. Maybe something more along the lines of caveats or upgrade notes?

Reply:
Yeah didn't quite feel right using that heading but threw it in for lack of better option at the time